tests/dbutils.py

```diff
@@ -29,10 +29,10 @@ def db_connection(dbname=None):
            "'%s'" % POSTGRES_USER)
 
 
-def create_db(dbname):
-    with db_connection().cursor() as cur:
+def create_db(dbname=TEST_DB_NAME):
+    with db_connection(dbname=None).cursor() as cur:
```

Comment:
Calling with explicit `None`, not `TEST_DB_NAME`, because test db not yet created